conf/application.conf

```diff
@@ -74,11 +69,10 @@ akka {
     # which may be located on another backend node. The node is selected 
     # by consistent hashing of the region id.
     /regionManager/router {
-      router = consistent-hashing
-      nr-of-instances = 1000
+      router = consistent-hashing-group
+      routees.paths = ["/user/regionManager"]
       cluster {
         enabled = on
-        routees-path = "/user/regionManager"
         allow-local-routees = on
         use-role = "backend-summary"
       }
```

Comment:
@patriknw Could you check that this configuration is correct? This was upgraded from Akka 2.3.x to 2.4.x, but the migration from old config in Akka 2.2.x was never done.